support-data-rest/src/test/java/org/uniprot/api/support/data/statistics/controller/StatisticsControllerIT.java

```diff
@@ -129,6 +188,39 @@ void getByReleaseAndTypeAndMultipleCategories() throws Exception {
                 .andExpect(jsonPath("$.results[1].items[1].entryCount", is(9457)));
     }
 
+    @Test
+    void getByReleaseAndMultipleCategories() throws Exception {
+        this.mockMvc
+                .perform(
+                        get(
+                                "/statistics/releases/2021_03?categories=EUKARYOTA,TOP_ORGANISM"))
+                .andDo(log())
+                .andExpect(status().isOk())
+                .andExpect(jsonPath("$.results.size()", is(2)))
+                .andExpect(jsonPath("$.results[0].categoryName", is("TOP_ORGANISM")))
+                .andExpect(jsonPath("$.results[0].searchField", is("sf Organism")))
```

Comment:
What is the searchField attribute means?

Reply:
It's not a part of this task. 
There's a column called 'search_field' in statistic_category table (for some categories not all).